src/js/api/urls.js

```diff
@@ -194,3 +194,4 @@ export const CYCLE_COUNT_ITEMS_BATCH = (locationId, cycleCountId) => `${CYCLE_CO
 export const CYCLE_COUNT_SUBMIT_COUNT = (locationId, cycleCountId) => `${CYCLE_COUNT(locationId)}/${cycleCountId}/count`;
 export const CYCLE_COUNT_SUBMIT_RECOUNT = (locationId, cycleCountId) => `${CYCLE_COUNT(locationId)}/${cycleCountId}/recount`;
 export const CYCLE_COUNT_REFRESH_ITEMS = (locationId, cycleCountId, removeOutOfStockItemsImplicitly) => `${CYCLE_COUNT(locationId)}/${cycleCountId}/refresh${removeOutOfStockItemsImplicitly ? '?removeOutOfStockItemsImplicitly=true' : ''}`;
+export const IMPORT_CYCLE_COUNT_ITEMS = (locationId) => `${CYCLE_COUNT(locationId)}/items/upload/count`;
```

Comment:
To match this constant with our previous naming for cycle count actions, I would call it `CYCLE_COUNT_IMPORT_ITEMS`